can-simple-observable.js

```diff
@@ -64,15 +64,18 @@ Object.assign(SimpleObservable.prototype, {
 	}
 });
 
-canReflect.assignSymbols(SimpleObservable.prototype, {
+var simpleObservableProto = {
 	"can.getValue": SimpleObservable.prototype.get,
 	"can.setValue": SimpleObservable.prototype.set,
 	"can.isMapLike": false,
 	"can.valueHasDependencies": function(){
 		return true;
-	},
-	//!steal-remove-start
-	"can.getName": function() {
+	}
+};
+
+//!steal-remove-start
+if (process.env.NODE_ENV !== 'production') {
+	simpleObservableProto[canSymbol.for("can.getName")] = function() {
```

Comment:
canSymbol.for() is not needed because this is passed into assignSymbols().

Reply:
Done, thanks!